data.tf

```diff
@@ -0,0 +1,59 @@
+data "aws_iam_policy_document" "dataworks_aws_collections_rehydration_write_data" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListBucket",
+    ]
+
+    resources = [
+      data.terraform_remote_state.common.outputs.published_bucket.arn,
+    ]
+  }
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "s3:Get*",
+      "s3:List*",
+      "s3:DeleteObject*",
+      "s3:Put*",
+    ]
+
+    resources = [
+      "${data.terraform_remote_state.common.outputs.published_bucket.arn}/pdm-dataset/*",
```

Comment:
Do you need delete on pdm, analytical dataset, entire data/ subdir?

Reply:
Good find, I planned to review it in next iteration, but as its sensitive I will update this shortly

Reply:
Delete and write access removed from all other location but dataworks-aws-collections-rehydration